include/deal.II/grid/connectivity.h

```diff
@@ -196,7 +196,7 @@ namespace internal
         if (d == 1)
           {
             static const dealii::ndarray<unsigned int, 3, 2> table = {
-              {{{0, 1}}, {{1, 2}}, {{2, 0}}}};
+              {{{0, 1}}, {{1, 2}}, {{0, 2}}}};
```

Comment:
This is unrelated to what you're doing, but this kind of thing should really be sitting in `ReferenceCell` as well.